Tasks/PackerBuildV1/src/packerHost.ts

```diff
@@ -101,12 +101,15 @@ export default class PackerHost implements definitions.IPackerHost {
         var installedPackerPath = tl.which("packer", false);
         var installedPackerVersion = this._getPackerVersion(installedPackerPath);
         console.log(tl.loc("InstalledPackerVersion", installedPackerVersion));
+        
+        var packerVersionString = this._taskParameters.packerVersionString ? this._taskParameters.packerVersionString : constants.CurrentSupportedPackerVersionString;
```

Comment:
lets also take this field into consideration if the custom template is selected

Reply:
done